src/main/java/com/manywho/services/email/email/EmailManager.java

```diff
@@ -22,9 +24,23 @@ public void send(ApplicationConfiguration configuration, Email email, boolean is
             sendAsynchronously = false;
         }
 
+        Mailer mailer = mailerFactory.createMailer(configuration);
+
         try {
-            mailerFactory.createMailer(configuration)
-                    .sendMail(email, sendAsynchronously);
+            AsyncResponse asyncTestConnection = mailer.testConnection(sendAsynchronously);
+            if (asyncTestConnection != null) {
+                asyncTestConnection.onException((e) -> {
+                    System.err.println("Connection error: " + e.getMessage());
```

Comment:
could you use the logger library that exist in the pom instead? it has many advantages.

Reply:
Will update this to use logger :+1: